docs/reference/indices/aliases.asciidoc

```diff
@@ -244,6 +244,65 @@ GET /alias2/_search?q=user:kimchy&routing=2,3
 // CONSOLE
 // TEST[continued]
 
+[float]
+[[aliases-write-index]]
+==== Write Index
+
+It is possible to associate the index pointed to by an alias as the write index.
+When specified, all index and update requests against an alias that point to multiple
```

Comment:
I think that if you explicitly set a the write flag to false, we shouldn't write even if it's the only index the alias points to.